.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
@adrian-livefront So is this `interval` is how often its checking for things like vulnerabilities? Do we know what the interval is right now?

Reply:
Interesting ok. I would think we might want this to be `daily`. Everytime there is a problem with `nokogiri` I've noticed that both the Livefront repositories and the completely separate SmartThings ones seem to get notified on the same day. That sort of suggests to me they might already be on a daily schedule.